README.md

```diff
@@ -1 +1,35 @@
-# ucw-app
\ No newline at end of file
+# Universal Connect Widget (Application)
+
+This repo is a full stack application which anyone can clone and self-host as a way to serve the connect widget via a url which can then be loaded into an iframe.
+
+#### Getting Started (in development)
+1. clone the repo
+2. copy .env.example
+3. Follow Initial setup for getting environment variables
+4. run `docker-compose up`
+5. load a widget url `http://localhost:8080/?job_type=agg&auth=${authCode}`
```

Comment:
I wonder if # 5-7 would make more sense in documentation about the server - something like APIs available through the server. Or maybe point to the example app for how to use `ucw-app`. I don't think they're technically needed to "Get started", but more of a how-to.

Reply:
Yeah good point. I'll update this.